src/visualization/visualization.cc

```diff
@@ -59,6 +65,7 @@ void ClearVisualizationMsg(VisualizationMsg& msg) {
   msg.points.clear();
   msg.lines.clear();
   msg.arcs.clear();
+  msg.messages.clear();
```

Comment:
I just realized that this member name is ambiguous. Could you please rename `messages` to `text_annotations` in VisualizationMsg?